cmake/modules/FindPugixml.cmake

```diff
@@ -0,0 +1,39 @@
+# - Find the native Pugixml includes and library
+
+find_path(Pugixml_INCLUDE_DIR 
+  NAMES
+    pugixml.hpp
+  PATHS
+    /usr/local/include
+    /usr/include
+)
+
+find_library(Pugixml_LIBRARY
+  NAMES
+    pugixml
+  PATHS
+    /usr/local/lib
+    /usr/lib
+)
+
+if (NOT Pugixml_INCLUDE_DIR OR NOT Pugixml_LIBRARY)
+  message(FATAL_ERROR "Couldn't find pugixml, you probably need libpugixml-dev or similar")
+endif()
+
+file(READ ${Pugixml_INCLUDE_DIR}/pugixml.hpp Pugixml_VERSION_CONTENT)
```

Comment:
pugixml does not install PC file that can be used with libfind_pkg_check_modules()?